debian/rules

```diff
@@ -11,6 +11,8 @@ extraopts += -DWITH_CEPHFS_JAVA=ON
 extraopts += -DCMAKE_INSTALL_LIBDIR=/usr/lib
 extraopts += -DCMAKE_INSTALL_LIBEXECDIR=/usr/lib
 extraopts += -DCMAKE_INSTALL_SYSCONFDIR=/etc
+NPROC=$(shell nproc)
```

Comment:
good call.  I'll rework tomorrow; all the lab slaves are offline for reasons I don't understand at the moment.